src/main/kotlin/inc/evil/d17map/dtos/ClassroomResponse.kt

```diff
@@ -10,5 +10,4 @@ data class ClassroomResponse(
     val equipmentIds: Set<UUID>,
     val id: UUID,
     val floorName: String,
-    val buildingName: String
```

Comment:
czemu?

Reply:
Bo robimy request na URI /buildings/{buildingName} I pisaliśmy na grupce, że to pole jest niepotrzebnie.

Trochę jakby wysłać zapytanie na GET /userId/{id usera}

I w odpowiedzi dostać {id usera}

Reply:
oki, ma to sens teraz